PlayTogether/PlayTogether/Sources/Onboarding/ViewController/CreateMeetViewController.swift

```diff
@@ -0,0 +1,184 @@
+        $0.font = .pretendardBold(size: 14)
+    }
+    
+    private let titleTextField = UITextField().then {
+        $0.placeholder = "동아리명 입력"
+        $0.font = .pretendardRegular(size: 14)
+        $0.textColor = .ptBlack02
+        $0.layer.borderColor = UIColor.ptGray03.cgColor
+        $0.layer.borderWidth = 1.0
+        $0.layer.cornerRadius = 10
+        $0.autocorrectionType = .no
+        $0.addLeftPadding()
+    }
+    
+    private let noticeTitleLabel = UILabel().then {
+        $0.text = "1~15(공백포함) 이내 한글, 영문, 숫자 사용 가능"
+        $0.font = .pretendardMedium(size: 12)
+        $0.textColor = .ptGray02
+    }
+    
+    private let introduceLabel = UILabel().then {
+        $0.text = "한 줄 소개"
+        $0.textColor = .ptBlack01
+        $0.font = .pretendardBold(size: 14)
+    }
+    
+    private let introduceTextField = UITextField().then {
+        $0.placeholder = "한 줄 소개 입력(15자 이내)"
+        $0.font = .pretendardRegular(size: 14)
+        $0.textColor = .ptBlack02
+        $0.layer.borderColor = UIColor.ptGray03.cgColor
+        $0.layer.borderWidth = 1.0
+        $0.layer.cornerRadius = 10
+        $0.autocorrectionType = .no
+        $0.addLeftPadding()
+    }
+    
+    private lazy var nextButton = UIButton().then {
+        $0.setTitle("다음", for: .normal)
+        $0.setTitleColor(.ptGray01, for: .normal)
+        $0.titleLabel?.font = .pretendardSemiBold(size: 16)
+        $0.backgroundColor = .ptGray03
+        $0.layer.borderColor = UIColor.ptGray02.cgColor
+        $0.layer.borderWidth = 1.0
+        $0.layer.cornerRadius = 10
+        $0.clipsToBounds = true
+        $0.isEnabled = false
+        $0.rx.tap
+            .bind {
+                self.nextButtonDidTap()
+            }
+            .disposed(by: disposeBag)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        configureNavbar()
+    }
+    
+    private func configureNavbar() {
+        let backIcon = UIImage.ptImage(.backIcon)
+        let leftButtonItem = UIBarButtonItem(image: backIcon, style: .plain, target: self, action: #selector(backButtonDidTap))
+        navigationItem.leftBarButtonItem = leftButtonItem
+        navigationItem.leftBarButtonItem?.tintColor = .white
+    }
+    
+    @objc private func backButtonDidTap() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    private func nextButtonDidTap() {
+        print("DEBUG: NextButton did tap")
+    }
+    
+    private func limitText(_ textField: UITextField, _ maxNumber: Int) {
+        textField.rx.text.orEmpty
+            .map { $0.count <= maxNumber }
+            .observe(on: MainScheduler.instance)
+            .subscribe(onNext: { isEditable in
+                if !isEditable {
+                    textField.deleteBackward()
+                }
+            }).disposed(by: disposeBag)
```

Comment:
백그라운드 스레드에서 돌아갈 로직이 없으면 observe(on:)이랑 subscribe 대신에 그냥 간단하게 drive를 써도 좋을 것 같아요!

Reply:
아래와 같이 코드를 바꿔보았는데 한 가지 문제가 생겼어

```swift
private func limitText(_ textField: UITextField, _ maxNumber: Int) {
        textField.rx.text.orEmpty.asDriver()
            .map{ $0.count <= maxNumber }
            .drive(onNext: { isEditable in
                if !isEditable {
                    textField.deleteBackward()
                }
            }).disposed(by: disposeBag)
}
```
- ⚠️ Reentrancy anomaly was detected.
  > Debugging: To debug this issue you can set a breakpoint in /Users/jiseok/Library/Developer/Xcode/DerivedData/PlayTogether-ambguzvgjrrxygacyyudiwahcgzi/SourcePackages/checkouts/RxSwift/Sources/RxSwift/Rx.swift:96 and observe the call stack.
  > Problem: This behavior is breaking the observable sequence grammar. `next (error | completed)?`
    This behavior breaks the grammar because there is overlapping between sequence events.
    Observable sequence is trying to send an event before sending of previous event has finished.
  > Interpretation: This could mean that there is some kind of unexpected cyclic dependency in your code,
    or that the system is not behaving in the expected way.
  > Remedy: If this is the expected behavior this message can be suppressed by adding `.observe(on:MainScheduler.asyncInstance)`
    or by enqueuing sequence events in some other way.


그래서 해결책으로 바꿔봤는데 코드 참조 부탁 ❗️